tests/query/v1/test_get_subgraph.py

```diff
@@ -0,0 +1,218 @@
+# --coding:utf-8--
+#
+# Copyright (c) 2020 vesoft inc. All rights reserved.
+#
+# This source code is licensed under Apache 2.0 License,
+# attached with Common Clause Condition 1.0, found in the LICENSES directory.
+
+from tests.common.nebula_test_suite import NebulaTestSuite
+from tests.common.nebula_test_suite import T_EMPTY, T_NULL
+import pytest
+
+class TestGroupBy(NebulaTestSuite):
+    @classmethod
+    def prepare(self):
+        self.load_data()
```

Comment:
You can `self.use_nba()`. no need to insert the data again. tests have inserted the nba data at the beginning.

Reply:
done